### DISCUSSION TITLE
go project initialisation error.

### DISCUSSION THREAD

**Discussion by nirmalsingh7950** [getgauge/gauge#1817] [General]
Given:
GOPATH: $HOME/go
Gauge version: 1.1.6
Commit Hash: 3af456f

Plugins
-------
go (0.1.4)
html-report (4.0.12)
js (2.3.15)
python (0.3.12)
screenshot (0.0.1)
xml-report (0.2.3)

I create go directory manually in my home directory .
then i set GOPATH to $HOME/go
Then Create src directory inside go directory
mkdir $HOME/go/src

**Create a project named test1**
cd $HOME/go/src/
mkdir test1
gauge init go
the above command gives me following error:
```
Error ----------------------------------

[Gauge]
Failed to initialize project. Cannot find Gauge template 'gauge-go'

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
	linux, 1.1.6, 3af456f
	go (0.1.4), html-report (4.0.12), js (2.3.15), python (0.3.12), screenshot (0.0.1), xml-report (0.2.3)
```

Can anyone please help me out how to resolve this issue

**Comment by zabil** [comment]
There's no official support for go lang at the moment. The template was removed from the list of templates. You could clone the following repo https://github.com/getgauge-contrib/gauge-example-go and build on it.

**Comment by ferdypruis** [comment]
I ran into this inconvenience as well, so I decided to recreate the template;
https://github.com/ferdypruis/gauge-template-go
